strings-csharp/CountNumberOfVowelsInString/CountNumberOfVowelsInString/VowelCounters.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Text.RegularExpressions;
+namespace CountNumberOfVowelsInString;
+
+public static partial class VowelCounters
+{
+    [GeneratedRegex("[AEIOUaeiou]")]
+    public static partial Regex regexVowels();
+
+    public static int CountVowelsUsingForLoop(ReadOnlySpan<char> sentence, ReadOnlySpan<char> vowels)
+    {
+        var total = 0;
+
+        for (var i = 0; i < sentence.Length; i++)
+        {
+            if (vowels.Contains(sentence[i]))
+            {
+                total++;
+            }
+        }
+
+        return total;
+    }
+
+    public static int CountVowelsUsingForEachLoop(ReadOnlySpan<char> sentence, ReadOnlySpan<char> vowels)
+    {
+        var total = 0;
+
+        foreach (var c in sentence)
+        {
+            if (vowels.Contains(c))
+            {
+                total++;
+            }
+        }
+
+        return total;
+    }
+
+    public static int CountVowelsUsingSwitchStatement(string sentence)
```

Comment:
For this method also, instead of string we can have it as ReadOnlySpan<char> sentence